model/manifest.go

```diff
@@ -50,6 +50,8 @@ type PluginSetting struct {
 	// "longtext" will result in a multi line string that can be typed in manually.
 	//
 	// "username" will result in a text setting that will autocomplete to a username.
+	//
+	// "custom" will result in a custom defined setting and will load the custom component registered for the webApp System Console.
```

Comment:
`webApp` or `webapp`?

Reply:
Or `Web App`? It seems to be referenced that way in docs right now: https://developers.mattermost.com/extend/plugins/webapp/reference/

Will updated.